client/onboarding/tasks/add-business-info-task/test/index.tsx

```diff
@@ -1,35 +1,45 @@
-/** @format */
-
 /**
  * External dependencies
  */
+import React from 'react';
 import { render, screen } from '@testing-library/react';
 import user from '@testing-library/user-event';
-import React from 'react';
+import { mocked } from 'ts-jest/utils';
 
 /**
  * Internal dependencies
  */
 import AddBusinessInfoTask from '../';
-import { useBusinessTypes } from 'wcpay/data/onboarding';
+import { useBusinessTypes } from 'onboarding/hooks';
+import { BusinessType } from 'onboarding/types';
```

Comment:
It looks like this is no longer being used in this file and can be removed!

Reply:
Good catch! Addressed in 8bd2f99